G2P.pm

```diff
@@ -40,56 +40,55 @@ limitations under the License.
 
  Options are passed to the plugin as key=value pairs, (defaults in parentheses):
 
- file            : path to G2P data file, as found at
-                   http://www.ebi.ac.uk/gene2phenotype/downloads
-
- af_monoallelic  : maximum allele frequency for inclusion for monoallelic genes (0.0001)
-
- af_biallelic    : maximum allele frequency for inclusion for biallelic genes (0.005)
-
- af_keys         : reference populations used for annotating variant alleles with observed
-                   allele frequencies. Allele frequencies are stored in VEP cache files. 
-                   Default populations are:
-                   ESP: AA, EA
-                   1000 Genomes: AFR, AMR, EAS, EUR, SAS 
-                   gnomAD exomes: gnomAD, gnomAD_AFR, gnomAD_AMR, gnomAD_ASJ, gnomAD_EAS, gnomAD_FIN, gnomAD_NFE, gnomAD_OTH, gnomAD_SAS 
-                   Separate multiple values with '&'
- af_from_vcf     : set value to 1 to include allele frequencies from VCF file. 
-                   Specifiy the list of reference populations to include with --af_from_vcf_keys    
- af_from_vcf_keys: reference populations used for annotating variant alleles with observed
-                   allele frequencies. Allele frequencies are retrieved from VCF files. If
-                   af_from_vcf is set to 1 but no populations specified with --af_from_vcf_keys
-                   all available reference populations are included. 
-                   TOPmed: TOPMed
-                   UK10K: ALSPAC, TWINSUK
-                   gnomAD exomes: gnomADe:AFR, gnomADe:ALL, gnomADe:AMR, gnomADe:ASJ, gnomADe:EAS, gnomADe:FIN, gnomADe:NFE, gnomADe:OTH, gnomADe:SAS
-                   gnomAD genomes: gnomADg:AFR, gnomADg:ALL, gnomADg:AMR, gnomADg:ASJ, gnomADg:EAS, gnomADg:FIN, gnomADg:NFE, gnomADg:OTH
-                   Separate multiple values with '&'
- default_af     :  default frequency of the input variant if no frequency data is
-                   found (0). This determines whether such variants are included;
-                   the value of 0 forces variants with no frequency data to be
-                   included as this is considered equivalent to having a frequency
-                   of 0. Set to 1 (or any value higher than af) to exclude them.
-
- types           : SO consequence types to include. Separate multiple values with '&'
-                   (splice_donor_variant,splice_acceptor_variant,stop_gained,
-                   frameshift_variant,stop_lost,initiator_codon_variant,
-                   inframe_insertion,inframe_deletion,missense_variant,
-                   coding_sequence_variant,start_lost,transcript_ablation,
-                   transcript_amplification,protein_altering_variant)
+ file                  : path to G2P data file, as found at http://www.ebi.ac.uk/gene2phenotype/downloads
```

Comment:
Picky comment: should use **https** instead of http

Reply:
I will update http to https on master. There are a lot of those in the file.